### PR TITLE
Morelogin

### PR DESCRIPTION
**Pull request by cbflanagan3**
Login/Register components with user routes. Still working on correct component redirect and persistence. Travis CLI error should be fixed. Default user created:  username: test password: password

### CODE REVIEW

**Review by JudithBLinder — CHANGES_REQUESTED** [review]
1. UserFunctions needs to be moved to userActions.js in actions folder
2. proptypes should be at the end of the file and not inside the render method. Cleaner...
3. Register is a really big file. needs to be broken down...

File: server/api/routes/users.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+const { User } = require('../../db/Models/User');
+
+userRouter.use(cors());
+userRouter.use(bodyParser.json());
+userRouter.use(bodyParser.urlencoded({ extended: true }));
+
+process.env.SECRET_KEY = 'secret';
+
+userRouter.get('/users', async (req, res) => {
+  const users = await User.findAll();
```
Comment (giantqtipz) [inline server/api/routes/users.js:15]: just curious, will we need to fetch all users at some point, or this is just a temporary code?

Comment (JudithBLinder) [reply server/api/routes/users.js:15]: We will need it for the friends list in friends component

**Review by JudithBLinder — APPROVED** [review]

**Review by shrutiJ154 — APPROVED** [review]
ok

**Pull request merged**